controllers/operatorconfig_controller.go

```diff
@@ -32,7 +33,7 @@ type ReconcileOperatorConfig struct {
 func (r *ReconcileOperatorConfig) SetupWithManager(mgr ctrl.Manager) error {
 	name := "config"
 	r.Client = mgr.GetClient()
-	r.log = ctrl.Log.WithName("controllers").WithName(strings.Title(name))
+	r.log = ctrl.Log.WithName("controllers").WithName(cases.Title(language.Und).String(name))
```

Comment:
Is this necessary given that we baseline on 1.17?

Reply:
CI seems running 1.18. [fail](https://ci.infinispan.org/blue/organizations/jenkins/Infinispan%20Operator/detail/PR-1717/2/pipeline/22)
The change seems to be safe, I would consider to keep it since 1.17 is eof [lifetime](https://endoflife.date/go)